packages/bpk-tokens/src/web/base/spacing.json

```
@@ -7,24 +7,52 @@
     "category": "spacings"
   },
   "props": {
+    "SPACING_FUNCTION_XS": {
+      "value": "@function bpk-spacing-xs() { @if global-variable-exists('new-spacing-values') and $new-spacing-values { @return .25rem; } @return {!SPACING_XS}; }",
```

Should this be something like
```suggestion
      "value": "@function bpk-spacing-xs() { @if global-variable-exists('new-spacing-values') and $new-spacing-values { @return {!SPACING_XS_NEW} } @return {!SPACING_XS}; }",
```